instructions.md

```diff
@@ -135,10 +135,15 @@ The required certificates are imported via secretGenerator on the `kustomization
 
 ### Step 3.2: Apply all manifests using kustomize
 
-By using the kustomization.yml we can now deploy the whole cluster in a single command.
+We are using the overlay feature of kustomize two create two variants: `eks` and `local-env`, in this guide we're using `eks`. (For a deployment on a local environment check the guide on `local-environment.md`)
```

Comment:
Same here, could you link `local-environment.md` to its source file?

Reply:
Fixed on 92edc7e